### PR TITLE
Input component

### PR DESCRIPTION
**Pull request by prath-dev**
Input Component Added .
Live Preview : http://127.0.0.1:5501/index.html#inputs

### CODE REVIEW

File: components/Inputs/inputs.css [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+    border: 2px solid orange;
+    color: black;
+}
+
+.error-input {
+    color: black;
+    width: 33rem;
+    height: 2rem;
+    margin-bottom: 1rem;
+    border: 2px solid red;
```
Comment (NandhiniKrishnamoorthy) [inline components/Inputs/inputs.css:51]: Try to use variables for colors

**Review by GeetaPogul — COMMENTED** [review]
good to merge

File: components/Inputs/inputs.css (hunk 1 above)
Comment (NandhiniKrishnamoorthy) [reply components/Inputs/inputs.css:51]: Try to add error comments also

**Review by NandhiniKrishnamoorthy — COMMENTED** [review]
Try to add error comment in UI

**Review by NandhiniKrishnamoorthy — APPROVED** [review]

File: components/Inputs/inputs.css (hunk 1 above)
Comment (prath-dev) [reply components/Inputs/inputs.css:51]: okkk

**Review by GeetaPogul — APPROVED** [review]

**Pull request merged**